### PR TITLE
Refs [Task][30578]: Create UI For Settings Screen

### PR DESCRIPTION
**Pull request by NguyenTaTrung**
## Related Tickets
- [#Task_30578: Create UI For Settings Screen](https://edu-redmine.sun-asterisk.vn/issues/30578)

## WHAT
- Design UI for Settings Screen, User Information Screen
- Design UI for Dialog User Information, BMI

## WHY
- Create layout to ready handle event click for Settings Screen

## Evidence (Screenshot or Video)
<p float="left">

</p>

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [x] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    override fun initData() {}
+
+    override fun initAction() {
+        buttonStart.setOnClickListener { openHomeFragment() }
+    }
+
+    private fun openHomeFragment() {
+        findNavController().navigate(UserInformationFragmentDirections.actionToNavigationHome())
+        (activity as MainActivity).showBottomNav()
```
Comment (huytq-2422) [inline app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt]: Sao không giao tiếp qua interface mà lại ép trực tiếp như này nhỉ

File: app/src/main/java/com/example/streetworkout/ui/dialog/bmi/DialogBmi.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import android.view.Window
+import com.example.streetworkout.R
+
+class DialogBmi(context: Context) : Dialog(context) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        requestWindowFeature(Window.FEATURE_NO_TITLE)
+        setContentView(R.layout.dialog_preview_bmi)
+
+        this.window?.setLayout(
```
Comment (datdt-1935) [inline app/src/main/java/com/example/streetworkout/ui/dialog/bmi/DialogBmi.kt]:
```suggestion
       window?.setLayout(
```

File: app/src/main/java/com/example/streetworkout/ui/dialog/information/DialogUserInformation.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.example.streetworkout.ui.dialog.information
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.example.streetworkout.R
+
+class DialogUserInformation : DialogFragment() {
```
Comment (datdt-1935) [inline app/src/main/java/com/example/streetworkout/ui/dialog/information/DialogUserInformation.kt]:
```suggestion
class UserInformationDialog : DialogFragment() {
```

File: app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt (hunk 1 above)
Comment (datdt-1935) [inline app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt]: dùng interface. Tìm hiểu Delegate Pattern nhé

Comment (NguyenTaTrung) [reply app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt]: Ok huy

Comment (NguyenTaTrung) [reply app/src/main/java/com/example/streetworkout/ui/user/UserInformationFragment.kt]: Dạ vâng a

**Review by datdt-1935 — APPROVED** [review]

**Pull request merged**